lib/sedna/README.md

```diff
@@ -1,2 +1,285 @@
 
-### Sedna Python SDK
+# Sedna Python SDK
+
+The Sedna Python Software Development Kit (SDK) aims to provide developers with a convenient yet flexible tool to write the Sedna applications.
+
+This document introduces how to obtain and call Sedna Python SDK.
+
+
+## Introduction
+
+Expose the Edge AI features to applications, i.e. training or inference programs.
```

Comment:
```suggestion
Expose the Edge AI features to applications, i.e. training or inference programs.
```
Expose the Edge AI features to applications(i.e., training or inference programs).

Reply:
accept